src/server.py

```diff
@@ -556,6 +559,46 @@ def run_dailyjob_query():
     return {'dailyjobs': output}
 
 
+# It's a temp endpoint to make database migration from alermanager server to heroku more easier
+@app.route("/data/dump/")
+@json_response
+def run_old_job_dump():
+    limit = request.args.get('limit')
+    offset = request.args.get('offset')
+    date_format = "%04d-%02d-%02d %02d:%02d:%02d"
+    db = create_db_connnection()
+    cursor = db.cursor()
+    query = "select slave, result, build_system_type, duration, platform, buildtype, " \
+            "testtype, bugid, branch, revision, date, failure_classification, failures " \
+            "from testjobs LIMIT %s OFFSET %s" % (limit, offset)
```

Comment:
Hi @jmaher, sorry for missing the comment for the endpoint.

So, the idea in here is:
we apply this endpoint in alert manager server, then we could run a simple script in heroku server and query data from alermanager server and insert them into the heroku.  And yes, the usage you mentioned above is exactly what we want. :)

However, I don't have a prefect way to ensure all the data can been migrated to heroku without any missing. Checkout and retransmission are too expensive or unnecessary I think. I think  good 'step length' can help us reduce the data missing, maybe 1000 or 500 data sets a time?